lib/twitter_cldr/js/compiler.rb

```diff
@@ -55,6 +56,34 @@ def compile_each(options = {})
           yield file, TwitterCldr.twitter_locale(locale)
         end
       end
+      def compile_test(options = {})
+        options[:minify] = true unless options.include?(:minify)
```

Comment:
I'm figuring out the best way to refactor this to not use duplicate code

Next up on my to-do list after RBNF is to split the data from the implementation. So, there will need to be a proper refactor of this bit of the code base. For simplicity and clarity of mind, I think I should fix this that time. Is that ok?